homeassistant/components/plex/sensor.py

```diff
@@ -88,39 +74,15 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 class PlexSensor(Entity):
     """Representation of a Plex now playing sensor."""
 
-    def __init__(
-        self,
-        name,
-        plex_url,
-        plex_user,
-        plex_password,
-        plex_server,
-        plex_token,
-        verify_ssl,
-    ):
+    def __init__(self, name, plex_url, plex_server, plex_token, verify_ssl):
         """Initialize the sensor."""
-        from plexapi.myplex import MyPlexAccount
-        from plexapi.server import PlexServer
-        from requests import Session
-
         self._name = name
         self._state = 0
```

Comment:
A note for the future: We should probably return `None` here instead to mean unknown state.